wasm/src/object.c

```diff
@@ -188,3 +436,30 @@ opa_value *builtin_object_union(opa_value *a, opa_value *b)
 
     return r;
 }
+
```

Comment:
ℹ️ With #3061 merged, we'll need an attribute here:
```suggestion

OPA_BUILTIN
```

Reply:
Done